web/tests.py

```diff
@@ -21,3 +21,32 @@ def test_basic_addition(self):
 True
 """}
 
+class WebViewTests(TestCase):
+    def test_loads_correctly(self):
+        """
+        Tests that the homepage loads correctly
+        """
+        response = self.client.get('')
+        self.assertEqual(response.status_code, 200)
+
+    def test_basic_html(self):
+        """
+        Tests that the title attribute is set to 'Psywerx', the page contains an element
+        with the class jumbatron, contains no empty paragraphs and that charset is utf-8
+        """    
+        response = self.client.get('')
+        html_no_space = response.content.replace(" ", "")
+        
+        self.assertIn('<title>Psywerx</title>', html_no_space)
+
+        has_jumbatron = False
+        if 'class="jumbotron"' in html_no_space or "class='jumbotron'" in html_no_space:
```

Comment:
A je res treba gleda obe moznosti? A imamo enkrat class="jumbotron" in drugic class='jumbotron'? Pomoje mas lahko samo en check.

Reply:
Ja tukaj sem mislil, da bi lahko kdaj nekdo mal spremenil izgled strani, pa bi on uporabljal drugačne narekovaje... Ampak ja res malo možnosti za to, pa hitro se da popravit